modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,243 +53,371 @@ and device ID 0x1015
-        "type": "host-local",
-        "subnet": "10.56.217.0/24",
-        "routes": [{
-          "dst": "0.0.0.0/0"
-        }],
-        "gateway": "10.56.217.1"
-      }
-    }'
+  resourceName: sriov <3>
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true" <4>
+  priority: 99 <5>
+  mtu: 9000 <6>
+  numVfs: 16 <7>
+  nicSelector:
+    vendor: "15b3" <8>
+    deviceID: "" <9>
+    pfName: ["eno3", "eno4"] <10>
+    rootDevices: ["0000:02:00.0", "0000:02:00.1"] <11>
+  deviceType: netdevice <12>
+  isRdma: false <13>
 ----
-+
-<1> `k8s.v1.cni.cncf.io/resourceName` annotation is set to `openshift.io/sriov`.
-<2> `type` is set to `sriov`.
+<1> The name of the CR.
+<2> The namespace of the CR, this must be in the same namespace of operator.
+<3> The resource name of SR-IOV device plugin. Prefix `openshift.io/` will be
+add when it's referred in pod annotation. It's allowed to create multiple CRs of `SriovNetworkNodePolicy` for one resource name.
+<4> The node selector to select which node to be configured. User can choose to
+label the nodes manually or with tools like Kubernetes Node Feature Discovery.
+Only SR-IOV network devices on selected nodes will be configured. And the SR-IOV
+CNI plugin and device plugin will be only deployed on selected nodes.
+<5> The priority of the policy,  the larger number gets lower priority.
+Range from 0 to 99.
+<6> The MTU of the virtual functions. Range from 1 to 9000. Leave it blank if
+you don't need to change the MTU.
+<7> The number of the virtual functions for each SR-IOV network device.
+<8> The vendor hex code of SR-IoV device. Allowed value "8086", "15b3".
+<9> The device hex code of SR-IoV device. Allowed value "158b", "1015", "1017".
+<10> The names of SR-IoV physical function.
+<11> The PCI addresses of SR-IoV physical function.
+<12> The driver type of the virtual functions. Allowed value "netdevice",
+"vfio-pci". Defaults to "netdevice".
+<13> The RDMA mode. Defaults to false.
 
-. Run the following command to create the `sriov-conf` CR:
-+
+== Configuring SR-IOV networks
+
+The SR-IOV Network Operator also introduces a CRD `SriovNetwork` for creating
+the Custom Resource of `NetworkAttachmentDefinition` of SR-IOV CNI plugin. When
+you create a CR of `SriovNetwork`, the operator will created a CR of
+`NetworkAttachmentDefinition` according.
+
+You should never have to modify this CRD. Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+You shall not modify or delete a Custom `SriovNetwork`, when it has been used by
+any running Pods.
+=====
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodePolicy`.
+
+[source,yaml]
 ----
-$ oc create -f sriov-conf.yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-conf <1>
+  namespace: sriov-network-operator <2>
+spec:
+  networkNamespace: default <3>
+  ipam: | <4>
+    {
+      "type": "host-local",
```

Comment:
`host-local` plugin is not official supported, might be good to user dhcp example?

Reply:
Fixed.